config/application.rb

```diff
@@ -70,6 +70,8 @@ class Application < Rails::Application
 
     config.feature_flag_mfa = ENV["FEATURE_FLAG_MFA"] == "enabled"
 
+    config.feature_flag_bypass_mfa = false
```

Comment:
Any value in making this an environment variable?
Might allow us to change the `govuk-docker` setup for testing without leaving unstaged code?

Then again, when we switch this on, i guess we'd just set this to true then eventually remove all this feature flag code?

Reply:
I think when we switch this on we'll just remove the flag entirely; it's only here really so that I can still have tests for the new functionality.